### PR TITLE
bump Default1pBlocking to 50% on release channel

### PR DESCRIPTION
**Pull request by antonok-edm**

### CODE REVIEW

**Review by pes10k — APPROVED** [review]
lgtm

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers.

File: seed/seed.json
Comment (iefremov) [inline seed/seed.json:44]: Wait, aren't we supposed to roll-out the `Disabled` state? 
The feature is enabled by default in the code @antonok-edm @pes10k

**Pull request merged**
